policyengine_uk/parameters/gov/hmrc/pensions/pension_contributions_relief_age_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: This is the age limit for pension contributions relief.
```

Comment:
```suggestion
description:  The pensions contributions relief is limited to filers at or above this age threshold. 
```

Reply:
change if not inclusive

Reply:
(Dont have to adjust everything here) But in general we try to make the descriptions as descriptive is possible, especially if there could be some ambiguity (e.g. inclusivity) we want to specify it here - especially for rates it is good to specify what rate is for -> this is what the user seems when clicking through the parameters in the app

Reply:
You make a good point. The exclusion is actually limited to those below this age limit, non-inclusive, so I will be clearer.